pkg/protocol/inactivity/inactivity.go

```diff
@@ -0,0 +1,88 @@
+package inactivity
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ipfs/go-log/v2"
+
+	"github.com/keep-network/keep-core/pkg/net"
+	"github.com/keep-network/keep-core/pkg/protocol/group"
+	"github.com/keep-network/keep-core/pkg/protocol/state"
+)
+
+// SignedClaim represents information pertaining to the process of signing
+// an inactivity claim: the public key used during signing, the resulting
+// signature and the hash of the inactivity claim that was used during signing.
+type SignedClaim struct {
```

Comment:
What about:
```suggestion
type SignedClaimHash struct {
```

I think this name better reflects the purpose. This is not a full claim object.

Reply:
Done in 0e3b9e5a5e7bfe2b010ecf77509bdfe5536713a2.